libraries/chain/hardfork.d/413.hf

```diff
@@ -1,4 +0,0 @@
-// #413 Add operation to claim asset fees
-#ifndef HARDFORK_413_TIME
-#define HARDFORK_413_TIME (fc::time_point_sec( 1446652800 ))
-#endif
```

Comment:
I think this commit is wrong. HARDFORK_413_TIME should not be removed, since it's being used somewhere else in the project.

Reply:
If we decide to remove the check about `HARDFORK_413_TIME`, we can remove this file as well.

Reply:
Dang, my fault .. I thought my dev has added 413 for me .. didn't realize he reused an existing varaible